src/poll.c

```diff
@@ -46,6 +46,16 @@
 #include "memptr.h"
 #include "pdu.h"
 
+#if ANDROID_VERSION >= 23
```

Comment:
nit: Something like this would be nice

```/* Number of sensor types supported by each Android version; used to explicitly avoid processing events from unknown or non-standard sensors. */